javascript-data-model/order-history.js

```diff
@@ -0,0 +1,87 @@
+var orderHistory = [
+  {
+    orderPlaced: 'August 4, 2020',
+    items: ['book'],
```

Comment:
This should be holding objects, with each object describing each individual item. For instance:
```
items: [
 {
   type: 'book',
   name: "The Art of Sql"
  }
]
```